### DISCUSSION TITLE
Where to go from here

### DISCUSSION THREAD

**Discussion by soomrofarrhan** [mrdbourke/pytorch-deep-learning#1052] [Q&A]
Hi. Hope you are doing well

I have gone through each lecture / concept explained by you in this course and I must say its wonderful. It has cleared my basic concepts regarding pytorch, model deployment etc. However, there are few things which I want to ask

1) Since this is 2020 course and its 2024 currently going on. As we know things get rapidly changed in market. So what should one opt or learn next to become in line with market expectation.

2) My goal was to become a freelancer. Please give me any guidance, advice or strategy that what should I go for next to achieve my goal keeping in view the current market trends / skills required as of today. If you provide any links of course or further material, I will be thankful to you. Thanks

**Comment by heschmat** [comment]
Hi there,
Have you checked the freelance pages to see what are the kinds of tasks they're asking in your areas of interest? 

P.S. the `issues` tab is not suitable for such discussions. The `discussions` tab would be better. The issues tab is for "issues" regarding codebase, like bug fixes & feature requests.

**Comment by mrdbourke** [comment]
Hi @soomrofarrhan ,

Thank you for the kind words about the course, I'm glad you enjoyed it :) 

If you've finished the course, I'd suggest similar to what is said above.

Start inspecting job postings + applying for them whilst also working on a few projects of your own.

For example, "Start the job before you have it". 

What I mean by this is that if you see a machine learning job/project out there that sparks your interest, start working on it so you can show someone what you've done already.

This will help for when applying for the role as well as become something you can start to add to your portfolio.

I've expanded a bit on these concepts in the following article/video:

* [Article] 5 free resources to help get a machine learning job - https://www.mrdbourke.com/get-a-machine-learning-job/
* [Video] 5 free resources to help you get a machine learning job - https://youtu.be/Al4bwR--BgY?si=j2YwPcx3nKuYQC6t
